packages/gatsby-source-wordpress/src/utils/graphql-queries.ts

```diff
@@ -74,7 +74,13 @@ export const introspectionQuery = /* GraphQL */ `
             }
           }
         }
-        fields {
+        # there might be interface in remote schema that seems to not deprecate the field
+        # but types implementing interface do deprecate the field
+        # and without asking to include deprecated - following errors happens
+        # Error: Interface field WpAcfPageFieldsFlexibleContentField_Layout.fieldGroupNa
+        # me expected but WpAcfPageFieldsFlexibleContentFieldFlexLayout1Layout_Fields do
+        # es not provide it.
```

Comment:
I assume something like prettier is forcing a specific line length here and splitting "does" and "Name" into two lines?

Reply:
I did not notice this being messed up - I think this is just because error I pasted was formatted like that in terminal (might have just very narrow terminal 😓 ) - we don't have formatting rules like that in gatsby

I pushed https://github.com/gatsbyjs/gatsby/pull/39263/commits/dce22d0f94a482ba93eca5db635aca07da8c050f to make it more readable